NuGet.Config

```diff
@@ -4,8 +4,4 @@
     <clear />
     <add key="nuget.org" value="https://api.nuget.org/v3/index.json" />
   </packageSources>
-  <config>
-    <add key="globalPackagesFolder" value="./packages" />
```

Comment:
>globalPackagesFolder [](http://example.com/codeflow?start=14&length=20)

Thanks for this. This folder is nearly 1GB of content in a standard build. The samples packages directory has 200MB. We use this pattern in other projects that have sarif-sdk as a submodule with the result that every enlistment is 4 - 5 GB of content, much of which is duplicated static packages.

Reply:
Yes, this also bit @scottoneil-ms and me, and wasted a considerable amount of our time.

![image](https://github.com/user-attachments/assets/b0de74ac-cf17-43d6-8bce-55835661adff)